### DISCUSSION TITLE
opendeck, unify and tripleplay

### DISCUSSION THREAD

**Discussion by kimgalibert** [shanteacontrols/OpenDeck#100] [Show and tell]
inside the guitar is an arduino mega2560.  having the controls right at hand completely changes the experience of playing midi guitar.  opendeck is amazing.

in the video: pitch bending via fsr, efx control via fsr, sound blending, sustain pedal and a bit of noodling around - 

https://vimeo.com/601439649

**Comment by paradajz** [comment]
That is just awesome. Very happy to see my project actually used. :) Thanks for the video!
